src/python/pants/backend/shell/target_types.py

```diff
@@ -42,7 +42,7 @@ class ShellSourceField(SingleSourceField):
     uses_source_roots = False
 
 
-class ShellGeneratingSourcesBases(MultipleSourcesField):
+class ShellGeneratingSourcesBase(MultipleSourcesField):
```

Comment:
@Eric-Arellano was the `..Bases` a typo, or should this be left as-is.

Reply:
Typo